### PR TITLE
remove limiter block in screen

### LINKED ISSUE #1290: screen limiter
Shouldn't we use an assert instead of this:

```
protected
  Modelica.Blocks.Nonlinear.Limiter limiter(uMin=0, uMax=1) 
    "Limits the control signal to avoid incorrect use by the user";
```
?

### PR DESCRIPTION
**Pull request by lucasverleyen**
Fixes #1290

### CODE REVIEW

**Comment by lucasverleyen** [comment]
@jelgerjansen @louisher Ready for review.
@Mathadon FYI

**Comment by lucasverleyen** [comment]
@jelgerjansen @louisher ready for review

File: IDEAS/Buildings/Components/Examples/HorizontalFinExample.mo [hunk 1]
```diff
@@ -63,8 +63,8 @@ public
     "Horizontal fin model with control input for displacement"
     annotation (Placement(transformation(extent={{64,20},{74,40}})));
   Modelica.Blocks.Sources.Sine sine(
-    amplitude=Modelica.Constants.pi/4,
-    offset=Modelica.Constants.pi/4,
```
Comment (jelgerjansen) [inline IDEAS/Buildings/Components/Examples/HorizontalFinExample.mo:67]: We should clearly mention that this is the reason why the reference results changed: `beta` used to go from 0 to pi/2, but by changing to the `Ctrl` input, `beta` is equal to `Ctrl*beta_max`, and `beta_max<pi/2`.

File: IDEAS/Buildings/Components/Shading/HorizontalFins.mo [hunk 2]
```diff
@@ -9,14 +9,15 @@ model HorizontalFins "Horizontal fin shading with 2 control input options"
...
-    "=true, to use input for fin inclination angle"
+    "=true, to use input for controlling the fin inclination angle. Set Ctrl=1 for fully closed shading."
     annotation(Evaluate=true);
   parameter Modelica.Units.SI.Angle beta(min=0) = 0
     "Fin inclination angle: 0 for horizontal inclination, see documentation"
     annotation (Dialog(enable=not use_betaInput));
 
   Real shaFrac "Shaded fraction of the glazing for direct solar irradiation";
   Real shaFracDif "Shaded fraction of the glazing for diffuse solar irradiation";
+  Modelica.Units.SI.Angle beta_max "Maximum fin inclination angle";
```
Comment (jelgerjansen) [inline IDEAS/Buildings/Components/Shading/HorizontalFins.mo:20]: This should be a (protected) parameter.

File: IDEAS/Buildings/Components/Shading/HorizontalFins.mo [hunk 3]
```diff
@@ -52,11 +55,12 @@ initial equation
     "In " + getInstanceName() + ": Either use_betaInput or use_displacementInput should be false.");
 
 equation
-
+  beta_max = acos(t/s);
```
Comment (jelgerjansen) [inline IDEAS/Buildings/Components/Shading/HorizontalFins.mo:58]: Include this under the parameter declaration of `beta_max`

File: IDEAS/Buildings/Components/Shading/HorizontalFins.mo [hunk 4]
```diff
@@ -30,6 +31,8 @@ protected
 
   Modelica.Blocks.Interfaces.RealInput beta_internal
     "Internal variable for inclination angle";
+  Modelica.Blocks.Interfaces.RealInput Ctrl_to_beta_internal
+    "Internal variable to linearly map the Ctrl input [0,1] onto the fin inclination angle [beta_min=0,beta_max]";
```
Comment (jelgerjansen) [inline IDEAS/Buildings/Components/Shading/HorizontalFins.mo:35]: I'm not sure if we need to introduce this new input, one of these should suffice I think.

File: IDEAS/Buildings/Components/Shading/HorizontalFins.mo [hunk 5]
```diff
@@ -30,6 +31,8 @@ protected
```
Comment (jelgerjansen) [inline IDEAS/Buildings/Components/Shading/HorizontalFins.mo:31]: `dispLim` can be removed due to the assert of `Ctrl` in `PartialShading`

**Review by jelgerjansen — COMMENTED** [review]
@lucasverleyen the changes look good, but the `HorizontalFins` model still needs some updates. I'll implement these.

File: IDEAS/Buildings/Components/Examples/HorizontalFinExample.mo (hunk 1 above)
Comment (jelgerjansen) [reply IDEAS/Buildings/Components/Examples/HorizontalFinExample.mo:67]: Mentioned here: https://github.com/open-ideas/IDEAS/issues/1290#issuecomment-2210442573

File: IDEAS/Buildings/Components/Shading/HorizontalFins.mo (hunk 2 above)
Comment (jelgerjansen) [reply IDEAS/Buildings/Components/Shading/HorizontalFins.mo]: Fixed in commit 7a0f9214d

File: IDEAS/Buildings/Components/Shading/HorizontalFins.mo (hunk 3 above)
Comment (jelgerjansen) [reply IDEAS/Buildings/Components/Shading/HorizontalFins.mo]: Fixed in commit 7a0f9214d

File: IDEAS/Buildings/Components/Shading/HorizontalFins.mo (hunk 5 above)
Comment (jelgerjansen) [reply IDEAS/Buildings/Components/Shading/HorizontalFins.mo:30]: Fixed in commit 13cab1fee

File: IDEAS/Buildings/Components/Shading/HorizontalFins.mo (hunk 4 above)
Comment (jelgerjansen) [reply IDEAS/Buildings/Components/Shading/HorizontalFins.mo:34]: There is an implementation possible where only two of the three internal inputs are needed (`beta_internal`, `Ctrl_to_beta_internal`, `disp_internal`), but that would make the model less readible for outsiders, so let's keep this as is.

**Pull request merged**